radio/src/opentx.cpp

```diff
@@ -1590,22 +1578,11 @@ void getADC()
 #endif
 
   DEBUG_TIMER_START(debugTimerAdcRead);
-  for (uint8_t i=0; i<4; i++) {
-    adcRead();
-    for (uint8_t x=0; x<NUMBER_ANALOG; x++) {
-      uint16_t val = getAnalogValue(x);
-#if defined(JITTER_MEASURE)
-      if (JITTER_MEASURE_ACTIVE()) {
-        rawJitter[x].measure(val);
-      }
-#endif
-      temp[x] += val;
-    }
-  }
+  adcRead();
   DEBUG_TIMER_STOP(debugTimerAdcRead);
 
-  for (uint32_t x=0; x<NUMBER_ANALOG; x++) {
-    uint16_t v = temp[x] >> (3 - ANALOG_SCALE);
+  for (uint8_t x=0; x<NUMBER_ANALOG; x++) {
+    uint16_t v = getAnalogValue(x) >> (1 - ANALOG_SCALE);
```

Comment:
This won't work if ANALOG_SCALE > 1. See also my later comments which are related to this.